provider/ecs/ecs_test.go

```diff
@@ -348,3 +348,36 @@ func TestTaskChunking(t *testing.T) {
 		}
 	}
 }
+
+func TestEcsGetBasicAuth(t *testing.T) {
+	cases := []struct {
+		desc     string
+		instance ecsInstance
+		expected []string
+	}{
+		{
+			desc:     "label missing",
+			instance: simpleEcsInstance(map[string]*string{}),
+			expected: []string{},
+		},
+		{
+			desc: "label existing",
+			instance: simpleEcsInstance(map[string]*string{
+				types.LabelFrontendAuthBasic: aws.String("user:password"),
+			}),
+			expected: []string{"user:password"},
+		},
+	}
+
+	for _, c := range cases {
+		c := c
```

Comment:
could rename `c` to `test`?

Reply:
If we rename `c` to `test` the complete file will be not consistent.

https://github.com/mmatur/traefik/blob/ce55834602fc20aba66a0ecf16302729c7d7da43/provider/ecs/ecs_test.go#L333

Reply:
you can rename all `c` 😃